lib/src/facts/resolvers/ruby_resolver.cc

```diff
@@ -80,14 +80,14 @@ namespace facter { namespace facts { namespace resolvers {
     {
         data rb_data;
 
-        auto const* ruby = api::instance();
-        if (!ruby || !ruby->initialized()) {
```

Comment:
We don't expect api::instance to throw an exception here, since Ruby should already have been initialized.